sierralocal/updater.py

```diff
@@ -135,14 +135,47 @@ def update_apobec():
         print('Could not update file for apobecs (apobecs.csv)\n'
               'Please download manually from https://github.com/hivdb/hivfacts/tree/main/data')
 
-
+def update_reference_fasta():
+    """
+    update reference fasta file for subtyper script
+    """
+    print("Downloading the latest subtype reference fasta file")
+    try:
+        latest = "https://cms.hivdb.org/prod/downloads/hiv-genotyper/genotype-references.fasta"
+        request = requests.get(latest)
+        filepath = os.path.join(mod_path, 'data', 'genotype-references.9c610d61.fasta')
```

Comment:
where does `9c610d61` come from?  If it is a checksum, shouldn't we be regenerating it based on the newly downloaded FASTA file?

Reply:
Its from the original file name that comes with the repository 5 years ago. I didn't want to mess with the file paths originally, but I have now changed it to just be `genotype-references.fasta`.